packages/cache-map/test/index.test.js

```diff
@@ -139,6 +139,96 @@ test('makeCacheMapKit (strong)', testMakeCacheMapKit, {
   keys: [{}, Symbol('unique symbol'), 'string key'],
 });
 
+test('makeCacheMapKit(1)', t => {
+  const { cache, getMetrics } = makeCacheMapKit(1);
+
+  const expectMetrics = { totalQueryCount: 0, totalHitCount: 0 };
+
+  const [key1, key2] = [{}, Symbol('unique symbol')];
+
+  const assertNoEntry = key => {
+    t.is(cache.has(key), false);
+    t.is(cache.get(key), undefined);
+    expectMetrics.totalQueryCount += 2;
+    t.deepEqual(getMetrics(), expectMetrics);
+  };
+  const assertEntry = (key, expectedValue) => {
+    t.is(cache.has(key), true);
+    t.is(cache.get(key), expectedValue);
+    expectMetrics.totalQueryCount += 2;
+    expectMetrics.totalHitCount += 2;
+    t.deepEqual(getMetrics(), expectMetrics);
+  };
+
+  assertNoEntry(key1);
+  assertNoEntry(key2);
+
+  // Populate up to capacity.
+  cache.set(key1, 'x');
+  expectMetrics.totalQueryCount += 1;
+  assertEntry(key1, 'x');
+
+  // Evict key1.
+  cache.set(key2, 'y');
+  expectMetrics.totalQueryCount += 1;
+  assertNoEntry(key1);
+  assertEntry(key2, 'y');
+
+  // Evict key2.
+  cache.set(key1, 'xx');
+  expectMetrics.totalQueryCount += 1;
+  assertNoEntry(key2);
+  assertEntry(key1, 'xx');
+
+  // Overwrite key1.
+  cache.set(key1, 'xxx');
+  expectMetrics.totalQueryCount += 1;
+  expectMetrics.totalHitCount += 1;
+  assertNoEntry(key2);
+  assertEntry(key1, 'xxx');
+});
+
+test('makeCacheMapKit(2)', t => {
```

Comment:
I don't think capacity 2 is any more special than 3+, but I'd love to see `0` as from what I can tell it's supported, but should have constant cache miss behavior.

Reply:
I missed that the test is titled `2` but it actually tests `0`.

Reply:
Fixed.